### ISSUE TITLE
Update puppeteer

### ISSUE THREAD

**Issue by paceaux** [paceaux/selector-finder#36]
when doing a search with `-c` we get a "puppeteer old headless deprecation warning"

Update puppeteer. 

![image](https://github.com/paceaux/selector-finder/assets/518862/a99dba9c-62aa-45f7-85af-435883d77141)

**Comment by paceaux** [comment]
WE should probably just update all the packages.

**Comment by paceaux** [comment]
Update:

Can't seem to get passed puppeteer 21.3.8. 

Something starting in puppeteer 21.4.0 gets me an error:

```bash

Jest: Got error running globalSetup - /Users/frtaylor/Repos/npm/selector-finder/test/config/setup.js, reason: Timed out after waiting 30000ms

```

**Comment by paceaux** [comment]
turns out puppeteer had a problem w/ environment variables. I deleted those, it all works fine. 

merged into dev under https://github.com/paceaux/selector-finder/pull/38

**Comment by paceaux** [comment]
merged into master under #42
